packages/procaptcha-react/src/components/theme.ts

```diff
@@ -11,30 +11,46 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import { createTheme } from '@mui/material'
 
-export const lightTheme = createTheme({
+const grey = {
+    0: '#fff',
+    100: '#f5f5f5',
+    200: '#eeeeee',
+    300: '#e0e0e0',
+    400: '#bdbdbd',
+    500: '#9e9e9e',
+    600: '#757575',
+    700: '#616161',
+    800: '#424242',
+    900: '#212121',
+}
+
+export const lightTheme = {
     palette: {
         mode: 'light',
         primary: {
-            main: '#1976d2',
-            contrastText: '#000',
+            main: '#542437',
```

Comment:
Are these equiv?

Reply:
No, I changed the colour to something other than blue. maybe I shouldn't have. Will set a poll in slack.